bin/minimal/node/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "minimal-node"
+version = "4.0.0-dev"
+description = "A fresh FRAME-based Substrate node, ready for hacking."
+authors = ["Substrate DevHub <https://github.com/substrate-developer-hub>"]
+homepage = "https://substrate.io/"
+edition = "2021"
+license = "MIT-0"
+publish = false
+repository = "https://github.com/substrate-developer-hub/substrate-node-template/"
+build = "build.rs"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[[bin]]
+name = "minimal-node"
+
+[dependencies]
+clap = { version = "4.0.9", features = ["derive"] }
+futures = { version = "0.3.21", features = ["thread-pool"] }
+futures-timer = "3.0.1"
+jsonrpsee = { version = "0.16.2", features = ["server"] }
+
+sc-cli = { path = "../../../client/cli" }
+sc-executor = {  path = "../../../client/executor" }
+sc-network = {  path = "../../../client/network" }
+sc-service = {  path = "../../../client/service" }
+sc-telemetry = { path = "../../../client/telemetry" }
+sc-transaction-pool = { path = "../../../client/transaction-pool" }
+sc-transaction-pool-api = { path = "../../../client/transaction-pool/api" }
+sc-consensus = { path = "../../../client/consensus/common" }
```

Comment:
Just thinking if there should be actual `substrate::client` and/or `substrate::primitives` namespaces?  
Re-exporting everything from the `substrate` crate could work, but not sure if its a good or horrible idea 😆